superbench/runner/playbooks/check_env.yaml

```diff
@@ -22,6 +24,8 @@
     container: sb-workspace
     sb_nodes: '{{ hostvars.values() | map(attribute="ansible_hostname") | sort }}'
     sb_env: |
+      # sb env
+      HOST_WS={{ ansible_user_dir }}/sb-workspace
```

Comment:
What's the function of this line?

Reply:
get the workspace path on the host (to change working directory when running)